snuba/settings/validation.py

```diff
@@ -95,3 +95,14 @@ def validate_settings(locals: Mapping[str, Any]) -> None:
         assert (
             physical_part < locals["LOCAL_SLICES"]
         ), f"physical slice for logical partition {logical_part} is {physical_part}, but only {partition_count} physical slices exist"
+
+    # build a set of sliced Kafka logical topics
+    sliced_topics = set()
```

Comment:
I feel like validations for slicing should start moving to their own function now. There's a lot of validations happening just for slicing and it becomes hard to reason about them in non-sliced cases

Reply:
Good point, done